lab_6_pipeline/data/opencorpora_tags_mapping.json

```diff
@@ -0,0 +1,56 @@
+{
+    "POS": {
+        "NOUN": "NOUN",
+        "NPRO": "PRON",
+        "ADJF": "ADJ",
+        "ADJS": "ADJ",
+        "COMP": "ADJ",
+        "ANUM": "ADJ",
+        "ADPRO": "ADJ",
+        "VERB": "VERB",
+        "INFN": "VERB",
+        "PRTF": "ADJ",
+        "PRTS": "ADJ",
+        "GRND": "VERB",
+        "NUMR": "NUM",
+        "ADVB": "ADV",
+        "PRED": "ADV",
+        "PREP": "ADP",
+        "CONJ": "CCONJ",
+        "PRCL": "PART",
+        "INTJ": "INTJ",
+        "UNKN": "X"
+    },
+    "Case": {
+        "nomn": "Nom",
+        "gent": "Gen",
+        "datv": "Dat",
+        "accs": "Acc",
+        "ablt": "Ins",
+        "loct": "Loc",
```

Comment:
show any evidence that this is a correct mapping (official docs, our description, etc.)

Reply:
fixed